temoa/extensions/template/components/example_limit.py

```
@@ -0,0 +1,62 @@
+    model: ExampleModel, r: Region, p: Period, t: Technology
+) -> ExprLike:
+    """Cap total new capacity built in period ``p`` for region/group ``r``/``t``."""
+    regions = geography.gather_group_regions(model, r)
+    techs = technology.gather_group_techs(model, t)
+
+    limit = value(model.example_new_capacity_limit[r, t])
+
+    new_cap_rtv = model.v_new_capacity
+    new_cap = quicksum(
+        new_cap_rtv[_r, _t, _v]
+        for _r, _t, _v in new_cap_rtv.keys()
```

Ruff (0.15.20)

[warning] Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)